react-sharing-state/my-app/src/Container.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+
+/**
+ * A container of items.
+ * One item is displayed at a time, with buttons to flip through them:
+ * - Next and Prev scroll forward and backward one item
+ * - An array of buttons scroll to a specific item
+ * TODO: The buttons don't work correctly!
+ */
+export default function Container({ items }) {
+  const [activeIndex, setActiveIndex] = useState(0);
+
+  return (
+    <div>
+      <div>{items[activeIndex]}</div>
+      <div>
+        <Button text="Prev" onClick={() => setActiveIndex(((activeIndex - 1) + items.length) % items.length)} />
+        <Buttons count={items.length} activeIndex={activeIndex} setActiveIndex={setActiveIndex} />
+        <Button text="Next" onClick={() => setActiveIndex((activeIndex + 1) % items.length)} />
+      </div>
+    </div>
+  );
+}
+
+/**
+ * A button that toggles its color between white and lightblue.
+ * TODO: Remove the toggle behavior and make the background color a prop, default white.
+ * TODO: When clicked, change the current item in the Container.
+ */
+function Button({ key, text, onClick, backgroundColor='white' }) {
+  return <button onClick={onClick} style={{ backgroundColor }}>{text}</button>;
+}
+
+/**
+ * An array of buttons.
+ * TODO: Highlight only the active button lightblue.
+ */
+function Buttons({ count, activeIndex, setActiveIndex}) {
```

Comment:
Name your event handler prop starting with `on`: `onActiveIndex`. That's just the React convention!